packages/dashboard/src/components.d.ts

```diff
@@ -130,13 +139,16 @@ declare namespace LocalJSX {
         "anchor"?: Anchor;
         "onResize"?: OnResize;
     }
+    interface ResizablePanes {
+    }
     interface TestingGround {
     }
     interface IntrinsicElements {
         "iot-dashboard": IotDashboard;
         "iot-dashboard-widget": IotDashboardWidget;
         "iot-selection-box": IotSelectionBox;
         "iot-selection-box-anchor": IotSelectionBoxAnchor;
+        "resizable-panes": ResizablePanes;
```

Comment:
best practice with webcomponents to have everything name spaced, since they are globally registered and collide (i.e. add the `iot-` prefix to all the components)

Reply:
Done!